teamworkApp/app/controllers/python_controller.rb

```diff
@@ -8,4 +8,9 @@ def studentGraph
         system 'python lib/studentGraph.py 1'
         redirect_to '/studentGraph/'
     end
+
+    def importData
+        system 'python lib/importData.py'
```

Comment:
Depending on whether Maeve's merge goes through before this, this call will no longer work because the python file will need an input for where the csv file lives. I don't know how this change will impact the button functioning properly.

Reply:
True, I can try and get my stuff reconciled Thursday morning. Unless you need this before then, if so I will work to get it done sooner.